src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1986,6 +1986,8 @@ Total\ items\ found\:=Total items found:
 Selected\ items\:=Selected items:
 Download\ linked\ online\ files=Download linked online files
 Select\ the\ entries\ to\ be\ imported\:=Select the entries to be imported\:
+Entry\ BibTeX\ data\:=Entry BibTeX data\:
+Show\ entry\ information=Show entry information
```

Comment:
It should be "Show BibTeX", because "information" is more general and could be more. For instance: citation data, is this entry already contained in the library, completeness of the bibtex data, ...

Reply:
done
<img width="366" alt="Zrzut ekranu 2024-01-22 o 21 49 03" src="https://github.com/JabRef/jabref/assets/58429636/7052db69-1a6d-4cfd-b164-f10adef394df">